srcpkgs/sov/template

```diff
@@ -0,0 +1,18 @@
+# Template file for 'sov'
+pkgname=sov
+version=0.63
+revision=1
+build_style=meson
+hostmakedepends="pkg-config cmake"
```

Comment:
You need `wayland-devel` here as well

Reply:
In hostmakedepends?

Reply:
Alright this seems to fix the failed checks